### PR TITLE
before add to new view, should properly remove from previous superview

### PR DESCRIPTION
**Pull request by chenyunguiMilook**
…erview

Fixes # 

**Type of change:** 

## Please check if the PR fulfills these requirements
- [ ] Self-review: I am confident this is the simplest and clearest way to achieve the expected behaviour
- [ ] There are no dependencies on other PRs or I have linked dependencies through Zenhub
- [ ] The commit messages are clean and understandable
- [ ] Tests for the changes have been added (for bug fixes / features)

### CODE REVIEW

File: UIKitTests/UIView/UIViewTests+subviews.swift [hunk 1]
```diff
@@ -90,6 +90,16 @@ class UIViewSubviewTests: XCTestCase {
         XCTAssertEqual(view.layer.sublayers!, [subview1.layer, subview2.layer, subview3.layer])
     }
 
+    func testAddSubviewShouldRemoveFromPreviousSuperview() {
+        let prevSuperview = UIView()
+        let subview = UIView()
+        prevSuperview.addSubview(subview)
+        let curtSuperview = UIView()
+        curtSuperview.addSubview(subview)
```
Comment (ephemer) [inline UIKitTests/UIView/UIViewTests+subviews.swift:98]: nit: could you change `curtSuperview` to `newSuperview`? We don't use that abbreviation elsewhere and it took me some time to realize what it's referring to

**Review by ephemer — APPROVED** [review]
Thanks @chenyunguiMilook. It would be great if you could update the naming in the test but otherwise looks fine to me

File: UIKitTests/UIView/UIViewTests+subviews.swift (hunk 1 above)
Comment (chenyunguiMilook) [reply UIKitTests/UIView/UIViewTests+subviews.swift:98]: sure

**Review by michaelknoch — APPROVED** [review]

**Comment by ephemer** [comment]
Hi @chenyunguiMilook, we just merged your PR, thanks a lot for your contribution

**Pull request merged**